web/web.config

```diff
@@ -123,7 +136,7 @@
         <add mimeType="*/*" enabled="false" />
       </staticTypes>
     </httpCompression>
-    <urlCompression doStaticCompression="true" />
+    <urlCompression doStaticCompression="false" doDynamicCompression="false" />
```

Comment:
Since we are blacklisting static files, maybe we can keep `doStaticCompression="true"` so loading times are smaller. It's important for performance (and some asset files, main js and css bundles, are large). Or there are still some static affected?

Reply:
yes, i agree